stories/bar/bar.stories.js

```diff
@@ -415,12 +447,13 @@ export const barAsFooter = () => `
 
 barAsFooter.parameters = {
     docs: {
-        iframeHeight: 350
+        iframeHeight: 150,
+        storyDescription: 'The footer bar should display only closing and finalizing action buttons that impact the entire page such as *Save* or *Cancel*. This bar is fixed at the bottom of the screen, and is not visible when scrolling up. It can be displayed by adding the <code>--footer</code> modifier class to the container element.'
```

Comment:
I'm assuming that it's fixed at the bottom of the screen, but need confirmation on that.